library/std/src/sys/windows/process/tests.rs

```diff
@@ -24,6 +24,31 @@ fn test_raw_args() {
     );
 }
 
+#[test]
+fn test_thread_handle() {
+    use crate::os::windows::io::AsRawHandle;
+    use crate::os::windows::process::{ChildExt, CommandExt};
+    const CREATE_SUSPENDED: u32 = 0x00000004;
+
+    let p = Command::new("cmd").args(&["/C", "exit 0"]).creation_flags(CREATE_SUSPENDED).spawn();
+    assert!(p.is_ok());
+    let mut p = p.unwrap();
+
+    extern "system" {
+        fn ResumeThread(_: *mut crate::ffi::c_void) -> u32;
+    }
+    unsafe {
+        ResumeThread(p.main_thread_handle().as_raw_handle());
+    }
```

Comment:
```suggestion
    extern "system" {
        fn ResumeThread(_: BorrowedHandle<'_>) -> u32;
    }
    unsafe {
        ResumeThread(p.main_thread_handle());
    }
```
Borrowed handle is safe to use in FFI so we can use it directly here.